Design styling forms

sulaimancode opened the pull request:
Added custom forms and styled them. Forms can take in additional fields(name) without having to change any css.

motri commented:
Text is not as readable as it could be, maybe a change of background color?

neoeno commented on app/assets/stylesheets/application.css:
```diff
@@ -13,3 +13,101 @@
+  border-bottom: 1px solid #c4c4c4;
+  border-radius: 5px;
+  box-shadow: 0 1px 5px rgba(0, 0, 0, 0.25);
+}
```
In general when styling it's best to add a class to the element and then use that to target styling — classes for styling, IDs (and sometimes other things) for Javascript targetting.

neoeno commented on app/assets/stylesheets/application.css:
```diff
@@ -13,3 +13,101 @@
+#user_email {
+  color:#666;
+  content:"You better do something";
+}
```
I think this `content` directive doesn't do anything, right?

neoeno commented on app/assets/stylesheets/application.css:
```diff
@@ -13,3 +13,101 @@
+}
+
+
+
```
A few blank lines here

neoeno commented on app/views/users/new.html.erb:
```diff
@@ -1,28 +1,25 @@
-      <%= form.text_field :name, :type => 'name' %>
+
+    <div class='text-field info'>
+      <%= form.text_field :name, :type => 'name', placeholder: 'Name..' %>
```
I'm a bit of a stickler for detail — but the rest of the fields have `...` and this one has `..` — probably best to stick to 3! There are a few instances of having just 2 throughout I believe?

neoeno commented on app/assets/stylesheets/application.css:
```diff
@@ -13,3 +13,101 @@
  *= require_tree .
  *= require_self
  */
+#new_user {
```
I'd break this out to another file. `application.css` can grow out of control and become terrible for merge conflicts if you're not careful.

neoeno commented on app/views/sessions/new.html.erb:
```diff
@@ -0,0 +1,24 @@
+    <%= link_to ("Sign up"), sign_up_path %>
+    <%= link_to ("Forgot password?"), new_password_path, id: 'forgot' %>
+  </div>
+<% end %>
```
This needs indenting another step.

sulaimancode commented:
Made all the required changes. Added an additional scss file for forms styling, used classes to target styling rather than ids, removed unnecessary lines of code and fixed some indentation.
